h/static/styles/partials/_search-result-sidebar.scss

```diff
@@ -102,6 +102,22 @@
   color: $grey-6;
 }
 
+.search-result-sidebar__logo-container {
+  display: flex;
+  align-items: center;
+  margin-bottom: 26px;
+  font-weight: bold;
+  text-transform: uppercase;
+  color: $group-organization-color;
+}
+
+.search-result-sidebar__org-logo {
+  height: 20px;
+  width: 20px; 
+  margin-right: 5px;
+  font-size: 6px;
```

Comment:
Was this `font-size` added to try to coax Chrome into displaying alt text more cleanly when images are busted? It feels a little bit forced to me, if so, as `6px` is not a font size that any (normal) human can read comfortably.

Reply:
Ok I have updated it to look like below. I think using 'Org:' is a good fit as it matches the rest of the formatting on the sidebar.
Firefox:
![image](https://user-images.githubusercontent.com/30059933/38709442-ec4fb602-3e6f-11e8-9770-ae90e9e6b13f.png)
Chrome:
![image](https://user-images.githubusercontent.com/30059933/38709452-0327fdda-3e70-11e8-9542-2bb25611ffe4.png)